nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -44,6 +46,7 @@
  * over time.
  */
 public abstract class NiFiProperties {
+    private static final Logger logger = LoggerFactory.getLogger(NiFiProperties.class);
```

Comment:
I understand introducing logging into this module may be undesirable. I feel that there are certain scenarios which are detected internal to logic here which dictate logging a warning, but if there are impactful side-effects, this can be removed.